lib/mods/replace-methods.js

```diff
@@ -8,6 +8,24 @@ const {
   getOriginal,
 } = require('../originals-helper.js');
 
+/**
+ *
+ * @param {Node} node - the node to examine
+ * @param {string} baseTypeName - the base type of the node
+ *
+ * @return {boolean} whether the method is a namespace method
+ */
+function isNamespaceMethod(node, baseTypeName) {
+  if (node.getExpression().getSymbol().getDeclarations()[0]
+      && node.getExpression().getSymbol().getDeclarations()[0]
+          .getInitializer) {
```

Comment:
This will always be true, because the function always exists. I think you want to test `.getInitializer()`.

However, the fact that this bug is passing the tests, means that either this second check is not important, or the tests are not thorough enough. I suggest deleting this second check for now, unless you have some test that will fail without it.

Reply:
So without the second check I'm getting the error:

```
TypeError: firstDeclaration.getInitializer is not a function
```

on the line

```
const callee = firstDeclaration.getInitializer().getText();
```

I don't think the function always exists if firstDeclaration does, which is why I have that second check

Reply:
Fascinating. What is firstDeclaration in that case?

Reply:
Seems to be an `InterfaceDeclaration` for `Math`